UI/hotkey-edit.cpp

```diff
@@ -58,6 +55,80 @@ void OBSHotkeyEdit::keyPressEvent(QKeyEvent *event)
 		break;
 #endif
 
+	/* Blacklist shortcuts */
+	case Qt::Key_Delete:
+		new_key.key = OBS_KEY_NONE;
+		break;
+	case Qt::Key_F11:
+		new_key.key = OBS_KEY_NONE;
+		break;
+	case Qt::Key_Left:
+		new_key.key = OBS_KEY_NONE;
+		break;
+	case Qt::Key_Right:
+		new_key.key = OBS_KEY_NONE;
+		break;
+	case Qt::Key_Shift:
+		new_key.key = OBS_KEY_NONE;
+		break;
+	case Qt::Key_Alt:
+		new_key.key = OBS_KEY_NONE;
+		break;
+	case Qt::Key_Control:
+		new_key.key = OBS_KEY_NONE;
+		break;
+	case Qt::Key_Space:
+		new_key.key = OBS_KEY_NONE;
+		break;
+	case (Qt::Key_V):
+		if (event->modifiers()==Qt::ControlModifier)
+			new_key.key = OBS_KEY_NONE;
+		break;
+	case (Qt::Key_C):
+		if (event->modifiers()==Qt::ControlModifier)
+			new_key.key = OBS_KEY_NONE;
+		break;
+	case (Qt::Key_E):
+		if (event->modifiers()==Qt::ControlModifier)
+			new_key.key = OBS_KEY_NONE;
+		break;
+	case (Qt::Key_R):
+		if (event->modifiers()==Qt::ControlModifier)
+			new_key.key = OBS_KEY_NONE;
+		break;
+	case (Qt::Key_F):
+		if (event->modifiers()==Qt::ControlModifier)
+			new_key.key = OBS_KEY_NONE;
+		break;
+	case (Qt::Key_S):
+		if (event->modifiers()==Qt::ControlModifier)
+			new_key.key = OBS_KEY_NONE;
+		break;
+	case (Qt::Key_D):
+		if (event->modifiers()==Qt::ControlModifier)
+			new_key.key = OBS_KEY_NONE;
+		break;
+	case (Qt::Key_Up):
+		if (event->modifiers()==Qt::ControlModifier)
+			new_key.key = OBS_KEY_NONE;
+		else
+			new_key.key = OBS_KEY_NONE;
+		break;
+	case (Qt::Key_Down):
+		if (event->modifiers()==Qt::ControlModifier)
+			new_key.key = OBS_KEY_NONE;
+		else
+			new_key.key = OBS_KEY_NONE;
+		break;
+	case (Qt::Key_Home):
+		if (event->modifiers()==Qt::ControlModifier)
+			new_key.key = OBS_KEY_NONE;
+		break;
+	case (Qt::Key_End):
+		if (event->modifiers()==Qt::ControlModifier)
```

Comment:
Code styling is inconsistent with the rest of the project (and with other lines of this commit) with this diff block.

1.) You want a space before and after operators (such as == in these cases).
2.) Surrounding the case values of the switch with parentheses is another inconsistency, and should generally be unnecessary.